### PR TITLE
Improve README.md for newcomers

### PR DESCRIPTION
**Pull request by MartinGuindon**
I've added a short _Getting Started_ section to the README.md to guide new users of dbt on how this package should be used with their project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+## Getting started
+
+To use dbt-utils:
+
+1. Reference the package in your dbt project by creating a `packages.yml` file, which should contain the following:
+
+  ```
+  packages:
+    - git: "https://github.com/fishtown-analytics/dbt-utils.git"
+      revision: master
```
Comment (drewbanin) [inline README.md]: we typically advise pinning your dep to a specific release. Can you change `master` to something like `0.1.22` and add a comment to consult https://github.com/fishtown-analytics/dbt-utils/releases for information about the latest release?

**Comment by drewbanin** [comment]
Thanks @MartinGuindon - this will be a great improvement! One small comment, and ignore the failing tests here

File: README.md (hunk 1 above)
Comment (MartinGuindon) [reply README.md]: Ah, _that's_ what you meant when you said "we need to bump it every time we make a new release". It makes total sense though. I'll update the PR.

For what it's worth, this recommendation is nowhere to be found in the [Package Management](https://docs.getdbt.com/docs/package-management) section. The example given there is pointing to master.

**Comment by MartinGuindon** [comment]
Do it!

>

**Comment by drewbanin** [comment]
Just updated the [package management](https://docs.getdbt.com/v0.13/docs/package-management) docs to include more info about revisions! Thanks again @MartinGuindon

**Pull request merged**